src/test/java/com/puppycrawl/tools/checkstyle/checks/ArrayTypeStyleCheckTest.java

```diff
@@ -55,7 +55,9 @@ public void testJavaStyleOn()
             "13:18: " + getCheckMessage(MSG_KEY),
             "19:44: " + getCheckMessage(MSG_KEY),
             "43:33: " + getCheckMessage(MSG_KEY),
+            "48:34: " + getCheckMessage(MSG_KEY),
             "48:36: " + getCheckMessage(MSG_KEY),
+            "54:27: " + getCheckMessage(MSG_KEY),
```

Comment:
We were missing violations for multidimensional array declarations after the first set of `[]`.